vocab_files/observable_property_concepts/disturbance.ttl

```diff
@@ -7,21 +7,20 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
 <https://linked.data.gov.au/def/test/dawe-cv/632e4c3c-0e73-4b3d-b1d6-7b38897da5e8>
     a skos:Concept ;
-    dcterms:source """McCallum K, Bignall J, Laws M, O’Neill S, Sparrow B. (unpublished draft) ‘Targeted Surveys module’ in ….. (eds) Ecological
-Field Monitoring Protocols Manual: Standardising environmental monitoring and data systems for improved decision
-making. Draft v 0.1 Report to DAWE. TERN, Adelaide.""" ;
+    dcterms:source "Ecological Field Monitoring protocols - Soil module, Targeted Surveys module draft v0.1, 30/11/2021" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:altLabel "land surface disturbance" ;
```

Comment:
I don't think having `land surface disturbance` as an alternative label makes sense. You're basically saying all instances of disturbance type is some kind of land surface disturbance, which is not true. Land surface is a specific type of survey (see Yellow Book) and disturbances can happen in many different types of surveys (e.g., vegetation, soil).